packages/v4/patternfly-docs/content/design-guidelines/components/spinner/spinner.md

```diff
@@ -7,40 +7,47 @@ section: components
 
 Spinners are used to demonstrate data loading. Spinners are offered in different sizes that follow the sizing of our icons. These sizes include extra small, small, medium and large. Depending on the size of information you are loading and the space you are dealing with, we recommend using the spinner size that generally matches.
 
+
 <img src="./img/spinner-sizes.png" alt="Visual of different spinner sizes" width="436"/>
 
+## Placement
+
+Position the spinner in the center of the element that is loading. To ensure proper background contrast, use a white or light background
+
+<img src="./img/spinner-full-page.png" alt="Example of spinner in full page" width="990"/>
+
 ### When to use
 Use a spinner:
+- when the user is advancing to a new page or step where the structure has not been loaded in yet.
 - when the data you are dealing with is unknown in both quantity and shape.
 - when in progress of loading a screen that may not have any data, for example an [empty state screen or failed outcome](/components/empty-state). 
```

Comment:
What do you mean by failed outcome here? If its a failed outcome then it should be an error state not a spinner.

Reply:
that was already here but I can change it if you think we need to

Reply:
for "when not to use" i added " When a loading screen does not have any data, use an [empty state screen or failed outcome](/components/empty-state) instead."